### PR TITLE
Feature/constraint operators

### PR DESCRIPTION
**Pull request by bolomcs50**

### CODE REVIEW

File: include/unleash/strategies/strategy.h [hunk 1]
```diff
@@ -12,6 +12,8 @@ struct Constraint {
     std::string contextName;
     std::string constraintOperator;
     std::vector<std::string> values;
+    bool inverted;
```
Comment (bernedom) [inline include/unleash/strategies/strategy.h]: initialize these with a sane default value
```
bool inverted{false};
bool caseInsensitive{false};
```

**Review by bernedom — CHANGES_REQUESTED** [review]

**Comment by bolomcs50** [comment]
@bernedom The thing with CI here is Windows passes, Ubuntu tests pass but the check fails because of some later coverage upload step fails and MacOS does not bass because during the tests it somehow can't parse a .json
We reproduced the failure to run tests on a Mac. The library itself compiles and works, though.
Do we merge anyways?

**Review by bernedom — APPROVED** [review]

**Pull request merged**